packages/geolocation/__test__/deletePlaces.test.js

```diff
@@ -0,0 +1,37 @@
+const boom = require('@hapi/boom');
+const { faker } = require('@faker-js/faker');
+const { deletePlace } = require('../src/index');
+
+describe('[DELETE] deletePlace method to delete in dababase', () => {
+  test('Given an id from a place, when the user wants to delete a place, then delete the place in the geolocation database with the Id and return the id', async () => {
+    const _id = '6257164a0c0f4e3a79cefe48';
```

Comment:
@CETHER what do you think if instead of hardcoding the id for the test here, we use faker to generate one?